packages/create-instance/create-scoped-slots.js

```diff
@@ -32,46 +32,45 @@ function getVueTemplateCompilerHelpers (): { [name: string]: Function } {
   names.forEach(name => {
     helpers[name] = vue._renderProxy[name]
   })
+  helpers.$createElement = vue._renderProxy.$createElement
   return helpers
 }
 
 function validateEnvironment (): void {
-  if (vueVersion < 2.5) {
-    throwError(`the scopedSlots option is only supported in ` + `vue@2.5+.`)
+  if (vueVersion < 2.1) {
+    throwError(`the scopedSlots option is only supported in ` + `vue@2.1+.`)
```

Comment:
`the scopedSlots option is only supported in ` + `vue@2.1+.`

↓

`the scopedSlots option is only supported in vue@2.1+.`